src/components/Layout/Header/NavBar/KeepKey/Modals/Wipe.tsx

```diff
@@ -9,7 +9,7 @@ import {
   ModalOverlay,
 } from '@chakra-ui/react'
 import { useToast } from '@chakra-ui/toast'
-import React, { useRef, useState } from 'react'
+import { useRef, useState } from 'react'
```

Comment:
Sweeeet. For all the React frens out here that might be confused with that removal: https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html#how-to-upgrade-to-the-new-jsx-transform
The JSX transform that was brought in React 17 removed the need for needing to have the default `react` export in scope when using react, therefore, these imports are useless nowadays

Reply:
And supported by TypeScript since 4.1 with the `"jsx": "react-jsx"` in our `tsconfig` 🚀